build-system/babel-config/pre-closure-config.js

```diff
@@ -41,6 +41,7 @@ function getPreClosureConfig() {
   const preClosurePlugins = [
     'optimize-objstr',
     argv.coverage ? 'babel-plugin-istanbul' : null,
+    './build-system/babel-plugins/babel-plugin-imported-helpers',
```

Comment:
Oh, let's add this to the `test` config, too.

Reply:
I believe this causes a problem when building tests:

```
[06:45:37] Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/circleci/project/node_modules/@babel/runtime/helpers/esm/objectWithoutPropertiesLoose.js
require() of ES modules is not supported.
```

https://app.circleci.com/pipelines/github/ampproject/amphtml/10188/workflows/ee1c1455-35d9-4332-8d04-4953cf08beb0/jobs/158380